sklearn/preprocessing/tests/test_common.py

```diff
@@ -113,31 +129,22 @@ def test_missing_value_handling(
             Xt_dense = est_dense.fit(X_train).transform(X_test)
             Xt_inv_dense = est_dense.inverse_transform(Xt_dense)
 
-        for sparse_constructor in (
```

Comment:
Here, I think it would be better to have a separate test for the sparse case. Otherwise, we are going to run the dense part for all the sparse containers instead of a single time.

Reply:
After checking again, we will need to have a dense transformer. So the best way is to still keep this `for` loop instead of parametrization (otherwise, we could use a pytest.fixture as well.)

Reply:
@work-mohit could you revert this part and use a `for` loop instead of a parametrization to gain some time in the testing.

Reply:
sorry, I just lost track, will fix now.

Reply:
@Charlie-XIAO can you help me with this ?

Reply:
Okay I will take a look soon. I’m not currently free but will come back in an hour or two.

Reply:
@work-mohit How would you want me to help? Note that if you want me to directly push some contents to this PR, I think you will need to grant me access to your fork (because I'm not maintainer).

Reply:
@Charlie-XIAO  Granted ! Have a look :)

Reply:
@work-mohit I'm so sorry I think I mistakenly pushed into one of your other branches. I will try to figure it out.

Reply:
@work-mohit I've updated this branch. I think the branch that I mistakenly pushed to is [fix/sklearn/linear_model/tests/test_common.py](https://github.com/work-mohit/scikit-learn/tree/fix/sklearn/linear_model/tests/test_common.py), but I'm not sure what's going on there. Could you please take a look?

Reply:
Okay sure! Thanks for helping!